lib/ansible/config/base.yml

```diff
@@ -1375,6 +1375,18 @@ HOST_KEY_CHECKING:
   ini:
   - {key: host_key_checking, section: defaults}
   type: boolean
+INVENTORY_ANY_ERRORS_FATAL:
```

Comment:
I would leave this for any failure during parsing (3rd option discussed in original ticket) and name this one INVENTORY_UNPARSED_SOURCE_IS_FAIL